### PR TITLE
Ansible uid gid mode good practices

### PR DESCRIPTION
**Pull request by leopaul36**

### CODE REVIEW

File: roles/kafka/broker/tasks/config.yml [hunk 1]
```diff
@@ -16,22 +17,34 @@
...
   vars:
     broker_id: "{{ groups['all'].index(inventory_hostname) }}"
 
 - name: Render jmxremote.password
   template:
     src: jmxremote.password.j2
     dest: "{{ kafka_broker_conf_dir }}/jmxremote.password"
+    owner: root
+    group: root
+    mode: "640"
```
Comment (gboutry) [inline roles/kafka/broker/tasks/config.yml:50]: Can Kafka read this file ?

Comment (leopaul36) [reply roles/kafka/broker/tasks/config.yml:50]: No and I realized I actually forgot to fix some actions as well for common and ranger. I will mark as draft and reopen once I fix everything.

Comment (gboutry) [reply roles/kafka/broker/tasks/config.yml:50]: It's still `owner: root, group: root, mode: 640`, which means kafka user cannot read this

File: roles/kafka/ranger/tasks/config.yml [hunk 2]
```diff
@@ -54,3 +57,6 @@
   template:
     src: core-site.xml.j2
     dest: "{{ kafka_broker_conf_dir }}/core-site.xml"
+    owner: root
+    group: root
+    mode: "644"
```
Comment (gboutry) [inline roles/kafka/ranger/tasks/config.yml:62]:
```suggestion
    mode: "644"

```

Missing a new line

**Review by gboutry — APPROVED** [review]

File: roles/kafka/ranger/tasks/config.yml [hunk 3]
```diff
@@ -54,3 +57,7 @@
...
+
```
Comment (rpignolet) [inline roles/kafka/ranger/tasks/config.yml:63]: Extra new line ? I think it was a comment from @gboutry

**Review by rpignolet — APPROVED** [review]

**Pull request merged**